src/sentry/integrations/slack/views/link_team.py

```diff
@@ -35,6 +42,8 @@
 SUCCESS_LINKED_TITLE = "Team linked"
 SUCCESS_LINKED_MESSAGE = "The {slug} team will now receive issue alert{workflow_addon} notifications in the {channel_name} channel."
 
+_logger = logging.getLogger(__name__)
```

Comment:
Why _? Other loggers I see in the codebase are just `logger`

Reply:
its a standard that Yash is trying to follow the convention of putting the `_` to emphasize that the logger is used within the file and shouldn't be exported. as part of the refactory, we have been removing references to common loggers in favor of this.